Launcher/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using System;
+using System.Threading;
+using QuantConnect;
+using QuantConnect.Lean.Engine;
+
+namespace Launcher
+{
+    class Program
+    {
+        /// <summary>
+        /// Primary Analysis Thread:
+        /// </summary>
+        public static void Main(string[] args)
+        {
+            QuantConnect.Lean.Engine.Engine.Launch();
```

Comment:
@jaredbroad Just started to make this change and I noticed that Config is static, so what would be the point of passing it to the Engine? In that case, it seems OK for the Engine to retrieve the values in the Constructor. Something like:

```
namespace Launcher
{
    class Program
    {
        public static void Main(string[] args)
        {
                var engine = new Engine();
                engine.Run();
        }
}
```

The other option is to get the configuration values in the Laucher.Main and passing LeanEngineSystemHandlers, LeanEngineAlgorithmHandlers and LiveMode to the Engine.
The problem I see with that is that the initialization code will have to be replicated in the application that uses the Engine DLL.

The third option involves many code changes: making the Config class non-static. But we will have to pass the Config object around to all the other classes that use it (ex. LiveTradingDataFeed).

What do you think?